configure.ac

```diff
@@ -821,33 +821,41 @@ AC_SUBST(LEMON)
 AM_CONDITIONAL([WITH_LEMON], [test "$lemon_available" = "yes"])
 
 # libedit
-AC_ARG_ENABLE(libedit,
-  [AS_HELP_STRING([--disable-libedit],
-    [use libedit for console. [default=auto-detect]])],
-  [enable_libedit="$enableval"],
-  [enable_libedit="auto"])
-if test "x$enable_libedit" != "xno"; then
-  m4_ifdef([PKG_CHECK_MODULES], [
-    PKG_CHECK_MODULES([LIBEDIT], [libedit >= 3.0],
-      [LIBS_SAVE="$LIBS"
-       LDFLAGS_SAVE="$LDFLAGS"
-       LDFLAGS="$LIBEDIT_LIBS $LDFLAGS"
-       AC_SEARCH_LIBS(el_wline, edit,
-                      [libedit_available=yes],
-                      [libedit_available=no])
-       LIBS="$LIBS_SAVE"
-       LDFLAGS="$LDFLAGS_SAVE"],
-      [libedit_available=no])
-    ],
-    [libedit_available=no])
-  if test "x$libedit_available" = "xyes"; then
-    AC_DEFINE(GRN_WITH_LIBEDIT, [1], [Use libedit with multibyte support.])
+libedit_available="no"
+AC_ARG_WITH([libedit],
+            AS_HELP_STRING([--with-libedit],
+                           [Support line editing (default: auto)]),
+            [LIBEDIT="$withval"],
```

Comment:
If we use `--with-XXX` style than `--enable-XXX` style, we should respect `$withval` to find libedit.
See https://github.com/groonga/groonga/blob/master/configure.ac#L1139-L1150 for example.